### PR TITLE
Practracker fixes

### PR DESCRIPTION
**Pull request by nmathewson**

### CODE REVIEW

File: scripts/maint/practracker/practracker.py [hunk 1]
```diff
@@ -164,6 +165,8 @@ def main(argv):
     parser = argparse.ArgumentParser(prog=progname)
     parser.add_argument("--regen", action="store_true",
                         help="Regenerate the exceptions file")
+    parser.add_argument("--list-overstrict", action="store_true",
```
Comment (teor2345) [inline scripts/maint/practracker/practracker.py]: I think this option should be called "list overbroad" or something.

Double negatives are hard, but I don't know how to eliminate this one. Here are some possible options:
* list excess(ive)
* list unnecessary
* list better (that is, we are doing better than our exceptions)

File: Makefile.am [hunk 2]
```diff
@@ -369,7 +369,9 @@ endif
 
 check-best-practices:
 if USEPYTHON
-	$(PYTHON) $(top_srcdir)/scripts/maint/practracker/practracker.py $(top_srcdir)
+	@if test "$$TOR_DISABLE_PRACTRACKER" = ""; then \
```
Comment (teor2345) [inline Makefile.am]: Can we implement this check in the practracker script instead?

The git hooks call practracker directly, not through the Makefile.

Comment (nmathewson) [reply Makefile.am]: Sure.  Fixed in 3221dc1b32f651

File: scripts/maint/practracker/practracker.py (hunk 1 above)
Comment (nmathewson) [reply scripts/maint/practracker/practracker.py]: 19536fd18d02a5 fixes this one.

**Pull request merged**